analytics/analytics-ui/src/app/table/table.component.html

```diff
@@ -1,9 +1,20 @@
-<mat-table [dataSource]="dataSource" class="mat-elevation-z8" matSort>
-  <ng-container [matColumnDef]="column" *ngFor="let column of tableColumns">
-    <mat-header-cell *matHeaderCellDef mat-sort-header>{{column}}</mat-header-cell>
-    <mat-cell *matCellDef="let element">{{element[column]}}</mat-cell>
-  </ng-container>
+<div style="margin-left:110px">
+  <h1>Table</h1>
+</div>
 
-  <mat-header-row *matHeaderRowDef="tableColumns; sticky: true"></mat-header-row>
-  <mat-row *matRowDef="let row; columns: tableColumns;"></mat-row>
-</mat-table>
\ No newline at end of file
+<!-- Dynamic Table -->
+
+<mat-card class="mat-elevation-z8">
+  <mat-table [dataSource]="dataSource" matSort>
+    <ng-container [matColumnDef]="column" *ngFor="let column of tableColumns">
+      <mat-header-cell *matHeaderCellDef mat-sort-header>
+        <h3>{{ column | uppercase }}</h3>
```

Comment:
Don't think we should uppercase the column names. Users should decide that for themselves.